### PR TITLE
Fixed nuget path

### PR DESCRIPTION
**Pull request by egarim**
Hi
the nuget path was wrong so I fixed it

### CODE REVIEW

**Comment by joergmetzler** [comment]
Please merge the PR. I had to build my own nuget package to apply the fix.
I would like to reference yours in the future to get other fixes as well.

File: MSBuild.Obfuscar/build/MSBuild.Obfuscar.targets [hunk 1]
```diff
@@ -14,11 +14,11 @@ WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and
...
+    <ObfuscatorToolsPath Condition="'$(NuGetProjectStyle)' == 'PackageReference'">$(MSBuildThisFileDirectory)..\..\..\obfuscar\2.2.26\tools</ObfuscatorToolsPath>
+    <ObfuscatorToolsPath Condition="'$(NuGetProjectStyle)' == 'PackageReference' And !Exists('$(ObfuscatorToolsPath)')">$(NuGetPackageRoot)obfuscar\2.2.26\tools</ObfuscatorToolsPath>
 
-    <ObfuscatorToolsPath Condition="'$(NuGetProjectStyle)' != 'PackageReference'">$(MSBuildThisFileDirectory)..\..\Obfuscar.2.2.26\tools</ObfuscatorToolsPath>
-    <ObfuscatorToolsPath Condition="'$(NuGetProjectStyle)' != 'PackageReference' And !Exists('$(ObfuscatorToolsPath)')">$(SolutionDir)packages\Obfuscar.2.2.26\tools</ObfuscatorToolsPath>
+    <ObfuscatorToolsPath Condition="'$(NuGetProjectStyle)' != 'PackageReference'">$(MSBuildThisFileDirectory)..\..\obfuscar\2.2.26\tools</ObfuscatorToolsPath>
+    <ObfuscatorToolsPath Condition="'$(NuGetProjectStyle)' != 'PackageReference' And !Exists('$(ObfuscatorToolsPath)')">$(SolutionDir)packages\obfuscar\2.2.26\tools</ObfuscatorToolsPath>
```
Comment (joergmetzler) [inline MSBuild.Obfuscar/build/MSBuild.Obfuscar.targets:21]: Not sure if this is correct. I think only the PackageReference style has another layout of the packages.

**Review by joergmetzler — CHANGES_REQUESTED** [review]

File: MSBuild.Obfuscar/build/MSBuild.Obfuscar.targets (hunk 1 above)
Comment (ennerperez) [reply MSBuild.Obfuscar/build/MSBuild.Obfuscar.targets:21]: https://www.nuget.org/packages/Obfuscar/

**Comment by warappa** [comment]
Hello @ennerperez ,
this fix was never merged. Unfortunately projects with PackageReference don't work (I just hit this). Fixing the targets file in my nuget-folder fixed the issue.
This PR (including the correction @joergmetzler pointed out) should be merged to fix this problem (lines 17-18 are right, 20-21 are not).
Thanks in advance!

**Pull request closed without merging**